### PR TITLE
:running: Update Go 1.13

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vincepri@vmware.com>

### CODE REVIEW

**Comment by vincepri** [comment]
Waiting on https://github.com/kubernetes/test-infra/pull/15620

**Comment by vincepri** [comment]
Waiting on https://github.com/kubernetes/test-infra/pull/15620

**Comment by vincepri** [comment]
/assign @detiber 

I'll follow-up with another PR for pprof

File: Dockerfile [hunk 1]
```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 # Build the manager binary
-FROM golang:1.12.10 as builder
+FROM golang:1.13.4 as builder
```
Comment (detiber) [inline Dockerfile]: Should this be 1.13.5 instead?

File: config/crd/bases/infrastructure.cluster.x-k8s.io_gcpclusters.yaml [hunk 2]
```diff
@@ -34,7 +34,6 @@ spec:
     listKind: GCPClusterList
     plural: gcpclusters
     singular: gcpcluster
-  preserveUnknownFields: false
```
Comment (detiber) [inline config/crd/bases/infrastructure.cluster.x-k8s.io_gcpclusters.yaml]: I don't think we actually want to remove this field.

File: config/crd/bases/infrastructure.cluster.x-k8s.io_gcpmachines.yaml [hunk 3]
```diff
@@ -37,7 +37,6 @@ spec:
     listKind: GCPMachineList
     plural: gcpmachines
     singular: gcpmachine
-  preserveUnknownFields: false
```
Comment (detiber) [inline config/crd/bases/infrastructure.cluster.x-k8s.io_gcpmachines.yaml]: Same here, I think we want to keep this.

File: config/crd/bases/infrastructure.cluster.x-k8s.io_gcpmachinetemplates.yaml [hunk 4]
```diff
@@ -16,7 +16,6 @@ spec:
     listKind: GCPMachineTemplateList
     plural: gcpmachinetemplates
     singular: gcpmachinetemplate
-  preserveUnknownFields: false
```
Comment (detiber) [inline config/crd/bases/infrastructure.cluster.x-k8s.io_gcpmachinetemplates.yaml]: Same, I think we probably want to keep this.

File: config/crd/bases/infrastructure.cluster.x-k8s.io_gcpclusters.yaml (hunk 2 above)
Comment (detiber) [reply config/crd/bases/infrastructure.cluster.x-k8s.io_gcpclusters.yaml]: https://github.com/kubernetes-sigs/cluster-api/blob/546d56bea2b8cd9ac24d90ed7599d209dfe2ccc5/Makefile#L187

**Pull request merged**